Tasks/FileTransformV2/package.json

```diff
@@ -18,9 +18,12 @@
   "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
   "dependencies": {
     "@types/mocha": "2.2.48",
-    "@types/node": "6.0.68",
+    "@types/node": "10.17.0",
     "@types/q": "1.0.7",
     "q": "1.4.1",
     "webdeployment-common-v2": "file:../../_build/Tasks/Common/webdeployment-common-v2-2.0.0.tgz"
```

Comment:
Looks like 'webdeployment-common-v2' needs to be updated also - since it uses v2.8.0 of azure-pipelines-task-lib. Could you please take a look?

Reply:
@agrataru I think it makes sense to publish an updated version to NPM to avoid affecting other tasks. Here is a [PR example](https://github.com/microsoft/azure-pipelines-tasks/pull/14049).

Reply:
@anatolybolshakov  @egor-bryzgalov added the following PR for updating `webdeplyment-common-v2` https://github.com/microsoft/azure-pipelines-tasks/pull/14098

Reply:
@anatolybolshakov @egor-bryzgalov the corresponding PR for webdeplyment-common-v2 was merged and the packaged was published to NPM.

Reply:
Thank you!